CognitoBot/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
```

Comment:
Why do we have a weatherforecast controller

Reply:
I was using it as a test bed for the sentiment analysis class